### DISCUSSION TITLE
Goblin Balloon

### DISCUSSION THREAD

**Discussion by BlazinMazon** [sunflower-land/sunflower-land#2248] [Ideas]
**Fragmented Purchasing:** This idea aims to focus on fragmented purchasing let's say for example I list 500 gold for sale, someone wants to buy 10, they can, and I now have 490 gold for sale. This will affect the fees per transaction which is why I've also brought up fee prices and trades per day. 

**Balloon Fee:** If this fragmented purchasing became available paying .30 cents if I want 10 kale is not viable. We would need a system that fluctuates with quantity purchased based off % maybe of total value of SFL. 10 kale .01 cent fee 100 .10 cent. All subject to change.

Trades Per Day: If these changes went into effect, the seller should only use only 1 trade per day and fragmenting the total should not affect the sellers remaining trades. However, the buyer would deplete per trade so perhaps a more trades per day or a system where we can purchase trade tickets like the block bucks.

**Comment by brian-man** [comment]
You might want to look at this:
#2162
